.circleci/continue_config.yml

```diff
@@ -1182,6 +1165,16 @@ jobs:
             yarn install --frozen-lockfile
             yarn build
 
+  verify-spec-snapshot:
+    working_directory: ~/openlineage/spec
+    docker:
+      - image: cimg/python:3.8
+    steps:
+      - *checkout_project_root
+      - run:
+          name: Verify spec snapshot
+          command: ./verify-spec-snapshot-up-to-date.sh
```

Comment:
There's already pre-commit hook for that. What additional verification does this job do?

Reply:
you're right. precommit script is no longer run in docker.